test.sh

```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
 
-set -e -x
+set -e
 
+export PATH=./bin:$PATH
 export RAILS_ENV=test
 export BUNDLE_WITHOUT=development
 
-bundle install --quiet
-bundle exec bowndler install
-bundle exec brakeman -q --no-pager --ensure-latest
-bundle exec rubocop .
```

Comment:
how come these are being removed from the test run?

Reply:
Sorry, yes they were removed because I was having issues in Go. Should be put back, just running test through the pipelines again to see if it goes through now

Reply:
@willhall88 Updated to include both `brakeman` and `robucop` tests